.github/workflows/test-android.yml

```diff
@@ -43,9 +43,9 @@ jobs:
           sudo apt-get install -y libgtk-3-dev webkit2gtk-4.1
 
       - name: setup node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
Please adjust this to node@18. We would like to run the tests on the oldest LTS/maintenance release. That being said, if there is an alias for it, it would be nice to not have to change it.